src/server.js

```diff
@@ -1,26 +1,49 @@
-// BASE SETUP
-var express = require('express');
-var bodyParser = require('body-parser');
-var mongoose = require('mongoose');
-var MONGO_PORT = 27017;
-mongoose.connect('mongodb://localhost:' + MONGO_PORT);
-var user = require('./users/user-router.js');
-var event = require('./Events/router.js');
-var report = require('./reports/report-router.js');
-var app = express();
-var port = process.env.PORT || 8080;
-
-app.use(bodyParser.urlencoded({ extended: true }));
+const express = require('express');
+const bodyParser = require('body-parser');
+const dotenv = require('dotenv');
+const mongoose = require('mongoose');
+
+// Routers.
+const user = require('./users/user-router.js');
+const event = require('./events/router.js');
+const report = require('./reports/report-router.js');
+const auth = require('./auth/auth-router.js');
+
+// load environment variables from .env file.
+dotenv.load({ path: process.env.ENV_PATH || '.env.example' });
```

Comment:
If we must have it hardcoded into `server.js` can the environment be something like `.env.default` instead?